dimod/core/composite.py

```diff
@@ -48,17 +48,25 @@
 import abc
 
 from dimod.core.sampler import Sampler
+from dimod.core.scoped import Scoped
 
 __all__ = ['Composite', 'ComposedSampler']
 
 
-class Composite(abc.ABC):
-    """Abstract base class for dimod composites.
+class Composite(Scoped):
+    """Abstract base class for dimod composites over scoped resources.
```

Comment:
" over scoped resources" --> should this be noted in the one-line summary of the class? Wouldn't it be a reasonable assumption? Seems to me this info is better positioned in line 62 or thereabouts.